app/components/Modals/VotingModal/VotingModal.jsx

```diff
@@ -0,0 +1,246 @@
+    votes: 444444444
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 555555555
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 666666666
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 777777777
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 888888888
+  },
+  {
+    address: 'AXmzKD3dvj7dPUQKkBeNZmRDYF6AhrwrtQ',
+    votes: 101010101
+  }
+]
+
+const votesAvailable = 123
+
+export default class VotingModal extends Component<Props, State> {
+  state = {
+    infoShowing: false,
+    confirmationShowing: false,
+    votes: {}
+  }
+
+  showInfoModal: Function
+  hideInfoModal: Function
+  handleNodeSelected: Function
+  handleCastVote: Function
+  showConfirmationModal: Function
+  hideConfirmationModal: Function
+  handleVoteSubmit: Function
+  hasVotes: Function
+
+  componentWillMount () {
+    this.showInfoModal = this.showInfoModal.bind(this)
+    this.hideInfoModal = this.hideInfoModal.bind(this)
+    this.handleNodeSelected = this.handleNodeSelected.bind(this)
+    this.handleCastVote = this.handleCastVote.bind(this)
+    this.showConfirmationModal = this.showConfirmationModal.bind(this)
+    this.hideConfirmationModal = this.hideConfirmationModal.bind(this)
+    this.handleVoteSubmit = this.handleVoteSubmit.bind(this)
+    this.hasVotes = this.hasVotes.bind(this)
+  }
+
+  render () {
+    const { hideModal } = this.props
+
+    return (
+      <BaseModal
+        title={textStrings.TITLE}
+        hideModal={hideModal}
+        style={{
+          content: {
+            width: '925px',
+            height: '700px'
+          }
+        }}
+      >
+        <div className={styles.modalContainer}>
+          <Button className={styles.infoButton} onClick={this.showInfoModal}>?</Button>
+          <div className={styles.contentContainer}>
+            <div className={styles.contentHeader}>{textStrings.TOP_10_CANDIDATES}</div>
+            <div className={styles.contentBody}>
+              <div className={styles.row}>
+                <div className={classNames(styles.leftCol, styles.title)}>
+                  {textStrings.CANDIDATES}
+                </div>
+                <div className={classNames(styles.rightCol, styles.title)}>
+                  {textStrings.NUMBER_OF_VOTES}
+                </div>
+              </div>
+              {this.renderNodeList()}
+            </div>
+          </div>
+          <Button
+            className={styles.submitButton}
+            onClick={this.handleCastVote}
+            disabled={!this.hasVotes()}
+          >{textStrings.CAST_VOTE}</Button>
+        </div>
+        {this.renderInformationModal()}
+        {this.renderVoteConfirmationModal()}
+      </BaseModal>
+    )
+  }
+
+  renderNodeList () {
+    return placehoderData.map((data, index) => {
+      const {address, votes} = data
+      return (
+        <div key={index} className={styles.row}>
```

Comment:
Could you use a more unique key? Its best when its actual ids, but if you use index then - perhaps `nodeRow${index}`